### ISSUE TITLE
IdsListView: Correct Listview Structure

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise-wc#2210]
**Describe the bug**

The listview has several structural issues with respect to aria and some inconsistencies

a) accessibility is incorrect (should be like https://main-enterprise.demo.design.infor.com/components/listview/example-index.html)
b) some items are direct in the DOM and some are slotted (need to see several different examples). This should be the same.
c) some items are wrapped in ids-draggable but this breaks accessibility (aria)

**To Reproduce**

1. Go to listview examples
2. Examine DOM in each
